### PR TITLE
Bug/claudia 5528 timestamp validation

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@jframos 

### DESCRIPTION
- fixing windowsize value before adding it to the redis queue.	
- fixing time stamp validation when windowsize is two

### CODE REVIEW

File: facts/server.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def process_request(request, tenantid, serverid):
     data.insert(len(fact_attributes) - 1, datetime.datetime.today().isoformat())
 
     # Check data coherency of time stamps
-    if len(mredis.range(tenantid, serverid)) > 2:
+    if len(mredis.range(tenantid, serverid)) >= 2:
```
Comment (flopezag) [inline facts/server.py]: MAGIC_NUMBER?

Why are you checking that it is gt or equal to 2?

Comment (geonexus) [inline facts/server.py]: because if it is only 1, y do not want to perform mredis.media

Comment (flopezag) [inline facts/server.py]: it needs at least <window_size> measurements to make the media, doesn't?

Comment (geonexus) [inline facts/server.py]: No, it needs the magic number 2.

**Pull request closed without merging**